homeassistant/components/command_line/sensor.py

```diff
@@ -35,6 +41,7 @@
         vol.Required(CONF_COMMAND): cv.string,
         vol.Optional(CONF_COMMAND_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
         vol.Optional(CONF_JSON_ATTRIBUTES): cv.ensure_list_csv,
+        vol.Optional(CONF_JSON_ATTRIBUTES_PATH): cv.string,
```

Comment:
We no longer allow integrations to add or change a platform YAML configuration.

More information on this can be found in Architecture Decision Record:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>

Reply:
So then this integration needs to be refactored so that it supports configuration under the domain key (command_line) while still supporting legacy configuration. Should that be its own PR? Command_line also needs to be refactored to use async, but I'm not sure I'm feeling that ambitious. Would the configuration refactor be accepted without also doing the async refactor?

Reply:
Somebody has already started a PR ([#63840](https://github.com/home-assistant/core/pull/63840)) to refactor command_line to the new config style and async, so converting this to draft for now in hopes that PR is eventually completed.